plugins/FlashSales/src/EasyAbp.EShop.Plugins.FlashSales.Application/EasyAbp/EShop/Plugins/FlashSales/FlashSalePlans/ProcessInvalidHashTokenCreateFlashSaleOrderCompleteEventHandler.cs

```diff
@@ -10,20 +10,20 @@
 
 namespace EasyAbp.EShop.Plugins.FlashSales.FlashSalePlans;
 
-public class RollBackInventoryCreateFlashSaleOrderCompleteEventHandler : IDistributedEventHandler<CreateFlashSaleOrderCompleteEto>, ITransientDependency
+public class ProcessInvalidHashTokenCreateFlashSaleOrderCompleteEventHandler : IDistributedEventHandler<CreateFlashSaleOrderCompleteEto>, ITransientDependency
```

Comment:
The naming is hard to understand. I am not entirely sure what it does, but can its name be such as `CreateFlashSaleOrderFailedEventHandler` or another?